### PR TITLE
dmt-359/quick-reference-user-checklist

### PR DESCRIPTION
**Pull request by murad-ali-MoJ**

### CODE REVIEW

File: source/documentation/tools/create-a-derived-table/quick-reference.md [hunk 1]
```diff
@@ -62,6 +68,90 @@ This list comprises everything you need to do and consider to get set up and rea
...
+<ul>
+    <li><code>sqlfluff lint .../path/to/sql/file.sql</code>, to lint a single SQL file</li>
+    <li><code>yamllint .../path/to/yaml/directory/</code>, to lint a whole directory of SQL files</li>
+</ul>
+
+##  Moving the model to Prod:
+If you're planning to submit a pull request for your models and subsequently merge them into the main repository, follow this recommended checklist:
+
+<ol type="1">
+  <li><b>Successful Deployment in Development: </b>Ensure that the development models successfully pass through the deploy-dev workflow. This step is crucial to guarantee that the model complies with linting and undergoes other essential checks.</li>
```
Comment (benwaterfield) [inline source/documentation/tools/create-a-derived-table/quick-reference.md:149]: May want to distinguish between deploying from local IDE and via Git PR. Ideally you'd want to make sure your models are working and linted locally before committing the changes and raising the PR (in which the deploy-dev is then a final check).
Also make sure the branch is up to date with main and any conflicts resolved.

Comment (tamsinforbes) [reply source/documentation/tools/create-a-derived-table/quick-reference.md:149]: good point; want to ensure people lint and run tests locally before raising the PR. The checklist should be in chronological/process order too - will make it easier to follow.

File: source/documentation/tools/create-a-derived-table/quick-reference.md [hunk 2]
```diff
@@ -62,6 +68,90 @@ This list comprises everything you need to do and consider to get set up and rea
...
+  <li><b>Comprehensive Testing:</b> Before moving forward to the production stage, ensure that the development models are accompanied by sufficient tests. Confirm the successful completion of these tests; it's a critical validation process.</li>
+  <li><b>Seamless User Experience: </b>Prepare the models for use by data consumers without introducing the risk of breaking changes. The user experience should remain smooth and uninterrupted.</li>
```
[the remark this answers is not in the document]
Comment (tamsinforbes) [reply source/documentation/tools/create-a-derived-table/quick-reference.md:151]: and link to the Data Modelling Concepts; some of these ideas may be useful there too.

**Review by tamsinforbes — APPROVED** [review]
LGTM - further iterations likely anyway

**Pull request merged**